tests/test_cyto_utils/test_cell_locations.py

```diff
@@ -26,8 +26,8 @@ def test_output_shape_and_required_columns(
     )
 
     # verify that the Nuclear_Location_Center_X and Nuclear_Location_Center_Y columns are present
-    assert "Nuclei_Location_Center_X" in cell_loc["CellCenters"][0][0].keys()
-    assert "Nuclei_Location_Center_Y" in cell_loc["CellCenters"][0][0].keys()
+    assert "Nuclei_Location_Center_X" in cell_loc["CellCenters"][0][0]
+    assert "Nuclei_Location_Center_Y" in cell_loc["CellCenters"][0][0]
```

Comment:
This is a pretty complicated test that I don't understand super well. Do you have a more specific code suggestion I could incorporate?

Reply:
Thanks for the reply! I felt similarly that this was a complicated test that I struggled to explain in terms of the change being made. I wondered if further clarification about the data type being leveraged here would make sense to help explain the changes being performed. I can also see why it might be friction-inducing to do this now.

Reply:
Thinking on this a bit more, I wonder if @gwaybio would have thoughts on this one. Specifically, I'd like to make sure we confirm the functionality remains the same after the changes.

Reply:
I did some digging in the code. The portion of the function that we're testing is this one: https://github.com/cytomining/pycytominer/blob/main/pycytominer/cyto_utils/cell_locations.py#L264
We're assembling a dataframe where we expect to find a dictionary that contains the two keys checked for. 

The reason this change was made is that checks `if x in dict.keys()` is redundant. You only need that pattern for `if x in dict.values()`The lint check wants us to use `if x in dict` instead. 

Because this is a lint check that applies to dictionaries and we are checking a dictionary the new language is identical to the old pattern.

Reply:
My understanding aligns with Ken's

Reply:
To clarify and ensure that we revisit this test if the dict type ever changes I added the following clarification:
```python
assert isinstance(cell_loc["CellCenters"][0][0], dict)
# verify that the Nuclear_Location_Center_X and Nuclear_Location_Center_Y dictionary keys are present
assert "Nuclei_Location_Center_X" in cell_loc["CellCenters"][0][0]
assert "Nuclei_Location_Center_Y" in cell_loc["CellCenters"][0][0]
```